taxcalc/reforms/TCJA_Reconciliation.json

```diff
@@ -0,0 +1,174 @@
+// Title: Tax Cuts and Jobs Act, Reconciliation verson
+// Reform_File_Author: Cody Kallen
+// Reform_Reference: http://docs.house.gov/billsthisweek/20171218/CRPT-115HRPT-466.pdf
+// Reform_Description:
+// -  New personal income tax schedule (regular/non-AMT/non-pass-through) (1)
+// -  New pass-through income tax schedule (2)
+// -  New standard deductions (3)
+// -  Repeal personal exemption (4)
+// -  Modification to child tax credit, nonrefundable dependent credits (5)
+// -  Modification of Alternative Minimum Tax exemption (6)
+// -  Repeal of certain above the line deductions (7)
+// -  Changes to itemized deductions (8)
+// -  Switch to chained CPI from CPI-U for tax paramter adjustment (9)
+// Reform_Parameter_Map:
+// - 1: _II_*
+// - 2: _PT_*
+// - 3: _STD
+// - 4: _II_em
+// - 5: _DependentCredit_*, _CTC_c, _CTC_ps, _ACTC_Income_thd
+// - 6: _AMT_rt*
+// - 7: _ALD_*
+// - 8: _ID_*
+// - 9: _cpi_offset
+// Note: _II_rt*, _PT_rt*, _STD and _II_em are rounded to the nearest integer value.
+{
+    "policy": {
+        "_II_rt1":
+            {"2018": [0.1],
+             "2026": [0.1]},
+        "_II_rt2":
+            {"2018": [0.12],
+             "2026": [0.15]},
+        "_II_rt3":
+            {"2018": [0.22],
+             "2026": [0.25]},
+        "_II_rt4":
+            {"2018": [0.24],
+             "2026": [0.28]},
+        "_II_rt5":
+            {"2018": [0.32],
+             "2026": [0.33]},
+        "_II_rt6":
+            {"2018": [0.35],
+             "2026": [0.35]},
+        "_II_rt7":
+            {"2018": [0.37],
+             "2026": [0.396]},
+        "_II_brk1":
+            {"2018": [[9525, 19050, 9525, 13600, 19050]],
+             "2026": [[11242, 22484, 11242, 16094, 22484]]},
+        "_II_brk2":
+            {"2018": [[38700, 77400, 38700, 51800, 77400]],
+             "2026": [[45751, 91502, 45751, 61242, 91502]]},
+        "_II_brk3":
+            {"2018": [[82500, 165000, 82500, 82500, 165000]],
+             "2026": [[110791, 184571,  92286, 158169, 184571]]},
+        "_II_brk4":
+            {"2018": [[157500, 315000, 157500, 157500, 315000]],
```

Comment:
Single/Separate -> 160000

Reply:
No, the bill specifies $157,500. See the link in the JSON, page 3 of the legislative text.

Reply:
Got it. Thanks.

Reply:
157,500 seems to be right according to http://docs.house.gov/billsthisweek/20171218/CRPT-115HRPT-466.pdf, page 2 and 3.